src/main/java/com/minecolonies/rankup/util/CommonUtils.java

```diff
@@ -50,4 +72,40 @@ public static String timeDescript(int timeNeeded)
         }
         return msg.toString();
     }
+
+    public static String dateNow()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        Date today = Calendar.getInstance().getTime();
+        return dateFormat.format(today);
+    }
+
+    @SuppressWarnings("squid:S1148")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1148' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1309)